assessmentModel/build.gradle.kts

```diff
@@ -85,16 +92,19 @@ kotlin {
                 implementation("org.jetbrains.kotlin:kotlin-test-annotations-common")
             }
         }
+        sourceSets["jvmMain"].dependencies {
+            implementation("org.jetbrains.kotlin:kotlin-test-junit")
+        }
         sourceSets["androidLibMain"].dependencies {
             implementation(libs.androidx.appcompat)
             implementation(platform("androidx.compose:compose-bom:${libs.versions.androidxComposeBom.get()}"))
             implementation(libs.androidx.compose.runtime)
         }
 
         // Set up dependencies between the source sets for Mac Silicon
-        val iosMain by getting
-        val iosSimulatorArm64Main by sourceSets.getting
-        iosSimulatorArm64Main.dependsOn(iosMain)
+//        val iosMain by getting
+//        val iosSimulatorArm64Main by sourceSets.getting
+//        iosSimulatorArm64Main.dependsOn(iosMain)
```

Comment:
So... what happens to the tests of encoding/decoding from Kotlin to Swift? (That's why I had set this up - so I could run tests on my Mac and as a GitHub action.)